### ISSUE TITLE
Underscore in file or path

### ISSUE THREAD

**Issue by tristelune1** [Strauman/exercisebank#55]
Hi,

My exercises are split in different files. For example I had:

- database_geometry.tex
- database_numbers.tex

I noticed there is a problem with the underscore. I had to make the following changes:

database_geometry.tex -> databaseGeometry.tex
database_numbers.tex -> databaseNumbers.tex

I had the same with paths. For example all my exercises are stored in:

exercises/geometry_euclidean/databaseGeometry.tex
exercises/numbers_integers/databaseNumbers.tex

I defined:

```
\exercisebanksetup{exercise
  directory=/home/user/Documents/exercises/,
part problems=Off}
```

So to insert some exercises:

\makeset[nohead]{myExerciseSet1}{\select{geometry_euclidean/databaseGeometry}{35,36,37,38}}
\makeset[nohead]{myExerciseSet2}{\select{numbers_integers/databaseNumbers}{78,79,80}}

After deleting the "_" in the path, there was no problem. I didn't read in the documentation, that it can be a problem. I don't know if it's intentional. For me there is no problem, but perhaps you should mention in in the documentation.

**Comment by Strauman** [comment]
Does it work if you escape the underscore? That is using `database\_geometry.tex`?

I might be able to fix this.

**Comment by tristelune1** [comment]
No, I get

`ERROR: Missing $ inserted. `

So it doesn't help to escape the underscore.

**Comment by Strauman** [comment]
Should be fixed in 95c0aff and be released with 0.2.3
